extensions/wikia/ImageReview/PromoteImageReview.setup.php

```diff
@@ -30,6 +30,7 @@
 if( function_exists('extAddBatchTask') ) {
 	extAddBatchTask($dir . "PromoteImageReviewTask.php", "promoteimagereview", "PromoteImageReviewTask");
 }
+//Possibly this line can be removed (as the usage new Wikia\\Tasks\\Tasks\\PromoteImageReviewTask was removed)

```

Comment:
Maybe we should add a note about this part of code to this ticket? It would be easier for them in the future to find the usages.